spec.md

```diff
@@ -1730,10 +1730,16 @@ message NodeStageVolumeRequest {
   // this capability. This is an OPTIONAL field.
   map<string, string> publish_info = 2;
 
-  // The path to which the volume will be published. It MUST be an
+  // The path to which the volume will be staged. It MUST be an
```

Comment:
Can you address Issue https://github.com/container-storage-interface/spec/issues/295#issuecomment-436455331 while you are touching this code?

Reply:
Done.